src/Model/Resource.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*****************************************************************************
+ *
+ *  PROJECT:     MTA PHP SDK
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        Resource.php
+ *  VERSION:     0.5
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+declare(strict_types=1);
+
+namespace MultiTheftAuto\Sdk\Model;
+
+use Exception;
+
+class Resource
+{
+    private $name;
+    private $server;
+
+    public function __construct($name, $server = null)
+    {
+        $this->name = $name;
+        $this->server = $server;
+    }
+
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    public function call($function, ...$arguments)
+    {
+        if (!$this->server) {
```

Comment:
Maybe make $server in constructor instead required?

Reply:
This is because getInput, static method of Mta class, use the constructor. That's why I made it optional